misc/fibonacci_formula.c

```diff
@@ -0,0 +1,32 @@
+/**
+ * \file
+ *
+ * \brief Implementation of [Fibonacci Sequence
+ * explicit formula solution](https://fabiandablander.com/r/Fibonacci.html)
+ *
+*/
+
+#include <math.h> /// for pow and sqrt
+#include <stdio.h>
+
+/*
+ * formula comes from quadratic formula and recurrence relation
+ * all the proof can be found in the related article in top of the file
```

Comment:
Missing parameter documentation.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable